tests/Fixer/Comment/CommentToPhpdocFixerTest.php

```diff
@@ -233,6 +233,26 @@ public function provideTestCases()
  * @ORM\Column(name="id", type="integer")
  */
 $bar = 42;
+EOT
+                ,
+            ],
+            [
+                <<<'EOT'
+<?php /* header comment */ $foo = true;
+
+/**
+ * This is my var
+ * @var string $foo
+ */
+$foo = 1;
+EOT
+                ,
+                <<<'EOT'
+<?php /* header comment */ $foo = true;
+
+// This is my var
+// /** @var string $foo */
```

Comment:
This isn't "nested", this is "commented out". This input should not change imho.

Reply:
It cannot be declined - current behaviour is not ok, [test case](https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/4038/commits/9e6c6b9b249260cf0fce966c022f6aaf673f7f4c) is failing due to [breaking](https://travis-ci.org/FriendsOfPHP/PHP-CS-Fixer/jobs/441227326) PHP syntax.

I'm open how to handle this - we may leave such a comment without change, we can change `*/` to `*-/` or remove both `/**` and `*/`

Reply:
> current behaviour is not ok, test case is failing due to breaking PHP syntax.

ok yeah I may have sounded harsher than intended, this is indeed not ok.

> we may leave such a comment without change

I think this should be the way to go here, as per my already given reason. Correct me if I'm wrong but the aim of this fixer is to fix comments that were intended (or used) as doc block in the first place, whereas this case is not, right?

Reply:
Good point that original comment was crashing the fixer, yet I believe we shall handle it differently

`// /** @var string $foo */`
is not intended to be documenting next structural element, as it's commented out.
I agree with @dmvdbrugge that handling shall be changed to not try to convert original comments into phpdoc comment, but instead we shall leave it unchanged.